AuthSamples/SwiftSample/ViewController.swift

```diff
@@ -156,6 +156,32 @@ final class ViewController: UIViewController, UITextFieldDelegate {
       view.endEditing(true)
   }
 
+  func verify(phoneNumber: String, completion: @escaping (PhoneAuthCredential?, Error?) -> Void) {
+    if #available(iOS 8.0, *) {
+      PhoneAuthProvider.provider().verifyPhoneNumber(phoneNumber) { verificationID, error in
+        guard error == nil else {
+          completion(nil, error)
+          return
+        }
+        let codeAlertController =
+            UIAlertController(title: "Enter Code", message: nil, preferredStyle: .alert)
+        codeAlertController.addTextField { (textfield) in
```

Comment:
Please be consistent on whether parentheses are used around parameters in closure. It looks like existing code doesn't when the return type is inferred. Same for line 174.